src/components/ConversationSettings/LockingSettings.vue

```diff
@@ -24,6 +24,9 @@
 		<div id="moderation_settings_lock_conversation_hint" class="app-settings-section__hint">
 			{{ t('spreed', 'Locking the conversation prevents anyone to post messages or start calls.') }}
 		</div>
+		<div v-if="hasCall" class="app-settings-section__hint">
+			{{ t('spreed', 'This will also terminate the ongoing calls') }}
```

Comment:
```suggestion
			{{ t('spreed', 'This will also terminate the ongoing call') }}
```

Reply:
damn, I thought I removed that s already...